### PR TITLE
Show message to inform users of topic split

### PR DESCRIPTION
**Pull request by jasonz-dfinity**
# Motivation

Inform users of the new topics and encourage them to set following.

# Changes

Add a message above the neuron table, and link to the related forum post. The changes are mostly based on a [previous PR](https://github.com/dfinity/nns-dapp/pull/4719)

# Tests

Unit tests and manual test
Screenshot 2024-08-05 at 9 30 06 PM

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/tests/lib/pages/NnsNeurons.spec.ts [hunk 1]
```diff
@@ -128,6 +134,18 @@ describe("NnsNeurons", () => {
...
       expect(await po.hasEmptyMessage()).toBe(true);
     });
+
+    it("should not render topic rename message", async () => {
+      const po = await renderComponent();
+
+      expect(await po.hasTopicRenameMessage()).toBe(false);
+    });
+
+    it("should not render topic rename message", async () => {
```
Comment (dskloetd) [inline frontend/src/tests/lib/pages/NnsNeurons.spec.ts:144]: Is there a difference between this test and the one above?

Comment (jasonz-dfinity) [reply frontend/src/tests/lib/pages/NnsNeurons.spec.ts]: No. Removed one of them. Nice catch!

File: frontend/src/lib/types/i18n.d.ts [hunk 2]
```diff
@@ -950,14 +950,14 @@ interface I18nAuth_neurons {
   title: string;
 }
 
-interface I18nAuth_proposals {}
+interface I18nAuth_proposals { }
```
Comment (dskloetd) [inline frontend/src/lib/types/i18n.d.ts:953]: Just curious, what caused these extra spaced?
I hope you didn't edit this file by hand?

**Review by dskloetd — APPROVED** [review]

File: frontend/src/lib/types/i18n.d.ts (hunk 2 above)
Comment (jasonz-dfinity) [reply frontend/src/lib/types/i18n.d.ts]: I did a workspace wide s/rename_topic/split_topic, and I think the IDE automatically tried to "format" every file it modifies. I think I need to look at my formatting setup again...

Comment (jasonz-dfinity) [reply frontend/src/lib/types/i18n.d.ts]: Fixed.

Comment (dskloetd) [reply frontend/src/lib/types/i18n.d.ts]: In any case, this file should be generated by `npm run i18n`.

**Pull request merged**